docs/src/pages/components/data-grid/components/components.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Data Grid - Components
+components: DataGrid, XGrid
+---
+
+# Data Grid - Components
+
+<p class="description">The grid is highly customizable. Override components using the `components` prop.</p>
+
+## Components prop
```

Comment:
Same feedback as for the virtualization second, h1 and h2 say the same thing. I think that it would be much better to flatten.

```suggestion
```